lib/rofl_ofdpa_fm_driver.cpp

```diff
@@ -1719,7 +1984,6 @@ cofflowmod rofl_ofdpa_fm_driver::add_bridging_unicast_vlan(uint8_t ofp_version,
 
   fm.set_command(OFPFC_ADD);
 
-  // FIXME do not allow multicast mac here?
```

Comment:
this comment is still valid, do not allow multi-cast mac when adding a unicast entry, see also the other one below

Reply:
Yeah, but now there is a specific multicast function available, so i do not think we have to specifically check for this anymore

Reply:
but the user could still try to set a unicast entry with a multicast MAC